src/app/sequence-editor/sequence-editor.component.html

```diff
@@ -2,9 +2,9 @@
   <div class="sequence-tab-left">
     <div class="sequence-card-list">
       <h3 class="sequence-section-title">All Cards</h3>
-      <div class="card-search-bar">Search for a card here...</div>
+        <input class="card-search-bar" #txt (input)="filterItems(txt.value)"type="search" [(ngModel)]="searchtext" id = "search-bar"placeholder="search" aria-label="search">
       <ol class="card-list">
-        @for( flashcard of _flashcards; track _flashcards){
+        @for( flashcard of filteredCards; track filteredCards){
```

Comment:
Try to keep line lengths under 100 characters. You can check in the bottom right of vs code
<img width="495" alt="image" src="https://github.com/PonderTeam/ponder-app/assets/4361319/9edceea8-1c68-4db3-b4e2-03ff9159d0bc">